### ISSUE TITLE
UserDefaults.has no longer working

### ISSUE THREAD

**Issue by justdan0227** [tbaranes/SwiftyUtils#137]
Did I miss where .has is no longer working or subscripts????

"Value of type 'UserDefaults' has no subscripts"

**Comment by tbaranes** [comment]
Both are still there and have unit tested passing, so it should be working. What version are you using? How are you installing it? Can you give some code that's not working? Any information could help to understand your issue, thanks :)

**Comment by justdan0227** [comment]
Thanks.. so I just did a pod update and now all of my userdefaults are telling me defaults[gDefaultsShoppingListKey] = dataToSend
value of type UserDefaults has no subscripts?

**Comment by justdan0227** [comment]
so is my coffee just not kicking in.  So I created a brand new project, and included SwiftyUtils in my pod file.  Installed, and then added this to my view did load

import UIKit
import SwiftyUtils

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        print ("Done")
        let defaults = UserDefaults.standard
        let dataToSend = "TEST"
        defaults["Key"] = dataToSend
    }
    
}

Building says same thing that UserDefaults has no subscripts?

**Comment by tbaranes** [comment]
Thanks for the explanation, I just tried with SwiftyUtils sample, it's working as expected. I'm gonna give it a try with CocoaPods.

**Comment by tbaranes** [comment]
@justdan0227 I definitely don't understand your issue, I just tried with a new project using CocoaPods, everything working. I added the project in attachment, let me know if it's working for you.

Some information, I did the test using:
- Xcode 11.4.1
- Cocoapods 1.9.1
- macOS 10.15.4 

[Test.zip](https://github.com/tbaranes/SwiftyUtils/files/4545489/Test.zip)

**Comment by tbaranes** [comment]
Ah, I've reproducing it. My local SwiftyUtils was 5.0.0 and not the latest version, seems like their an issue using the last version.

**Comment by justdan0227** [comment]
ah ha!  thanks.. how do I downgrade it for now

**Comment by tbaranes** [comment]
Working on a release right now, it should be available in 5min max :)
(if nothing goes wrong...)

**Comment by tbaranes** [comment]
Meanwhile, you can either target an old version in your Podfile:

```
    pod 'SwiftyUtils', '~> 5.0.0'
```

**Comment by justdan0227** [comment]
Thanks.. back on 5.0.0 for now and working

**Comment by tbaranes** [comment]
@justdan0227 I just released 5.2.0 which should fix the CocoaPods issue, I'm interesting to have your feedback on it to confirm the fix is real. To test it, just rollback to ` pod 'SwiftyUtils'` in your Podfile.

About the issue itself, I changed a few files arborescence in the 5.1.0 and forgot that CocoaPods doesn't embed the files automatically (like SPM / Carthage)... sorry for that, and thanks again for the report.
